examples/finetune/finetune.sh

```diff
@@ -4,16 +4,15 @@ cd ../..
 
 EXE="./finetune"
 
-MODEL="c:/models/openllama-3b-v2.gguf"
+# MODEL="c:/models/openllama-3b-v2-q8_0.gguf" # This is the model the readme uses.
+MODEL="c:/models/openllama-3b-v2.gguf" # An f16 model. Note in this case with "-g", you get an f32-format .BIN file that isn't yet supported if you use it with "main --lora" with GPU inferencing.
 
 while getopts "dg" opt; do
   case $opt in
     d)
       DEBUGGER="gdb --args"
       ;;
     g)
-      # GPU. The makefile doesn't support CUDA on Windows, so I have to use CMake and so main is built to a different location.
-      # Note: "-d" doesn't really work with this - it will run under gdb, but there are no debugging symbols (in a format gdb understands). I think the easiest workaround is to use WinDbg instead.
```

Comment:
It is possible to add debug generation flags, like:
-ggdb3 -Og -gdwarf-5.

For more legacy gdb versions, even running under windows, we could reduce to -gdwarf-4

Reply:
It looks like you're talking about gnu compiler flags. Your comments may be relevant to the 'LLAMA_DEBUG' section of the Makefile, which currently uses "-O0 -g".

The problem with CUDA is that we're using CMake, and MSVC compilers, which afaik are unable to produce DWARF info.